src/index.html

```diff
@@ -14,6 +14,7 @@
         <h1 class="txtMain">Awesome Books</h1>
         <div class="books-cont">

```

Comment:
Please make you README professional. Kindly add the project name, project description, and instructions to set the project up locally. Moreover, please add an active live link or remove it from README.